.gitignore

```diff
@@ -17,6 +17,7 @@ website/.bundle
 website/vendor
 
 packer-test*.log
+packer_cache
```

Comment:
this we can move to our global gitignore, but the test is supposed to clean this dir

Reply:
oh, I see that you removed the builder cleanup and that's why it wasn't cleaning up the folder for you

Reply:
Yeah - that was to get a working example :smile: